google/ggrc-core
# GGRC-5019 Use relationships for object mappings of Task group

# Issue description

Currently, task groups use specific way to load mapped objects. Need to use unified way of loading mapped objects via `relationships`.

# Steps to test the changes

1. Open any workflow
2. Go to any task group on "Setup" tab.
3. Map any objects to task group.

*Actual result*: FE uses POST `task_group_objects` to create mappings between task group and objects.
 
*Expected result*: Use relationships instead of `task_group_objects` request.
Almost all objects across the application use 'relationships' to perform such mappings.

# Solution description

Used `QueryAPI` to load mapped objects to some task group.

# Sanity checklist




# PR Review checklist
